CommandLine/Modular/Maven/hellofx/pom.xml

```diff
@@ -22,28 +22,28 @@
         <dependency>
             <groupId>org.openjfx</groupId>
             <artifactId>javafx-controls</artifactId>
-            <version>12.0.1</version>
+            <version>12.0.2</version>
```

Comment:
Use a maven property for this to make future changes easier.

Reply:
Even if we use Maven properties, these will be on top of each pom, so we well have to update every single file either way.

We could add a parent pom for all the maven samples, but I don't think we should, as each sample is isolated.

Reply:
Sure. However, in this case we update it at one place  instead of spreading it across the file.

Reply:
But this applies only if we have two or more dependencies, so it is a little bit overkilling imo

Reply:
ok